project/Dependencies.scala

```diff
@@ -44,7 +44,7 @@ object Dependencies {
     val camelCore = "org.apache.camel" % "camel-core" % "2.17.7" exclude ("org.slf4j", "slf4j-api") // ApacheV2
 
     // when updating config version, update links ActorSystem ScalaDoc to link to the updated version
-    val config = "com.typesafe" % "config" % "1.3.2" // ApacheV2
+    val config = "com.typesafe" % "config" % "1.3.3" // ApacheV2
```

Comment:
I assume that means it is binary backwards compatible

Reply:
yeah, I guess

Reply:
Yes it is :)